readme.md

```diff
@@ -191,6 +242,24 @@ Execute a file synchronously.
 
 Returns or throws a [`childProcessResult`](#childProcessResult).
 
+### \$(options?)\`command\`
+
+Same as [`execa()`](#execafile-arguments-options) except both file and arguments are specified in a single tagged template string. For example, `` $`echo unicorns` `` is the same as `execa('echo', ['unicorns'])`.
+
+The [`shell` option](#shell) must be used if the `command` uses shell-specific features (for example, `&&` or `||`), as opposed to being a simple `file` followed by its `arguments`.
+
+The return value of this function depends on whether you are binding options or executing a command:
+- If you are binding options, for example with `const $$ = $(options)`, the return value is a new `$` instance with the options bound to both the asynchronous `$` and synchronous `$.sync` APIs. It's important to note that consecutive calls to this API will shallow merge the options.
+- If you are executing a command, the result is a `Promise` that resolves or rejects with a [`childProcessResult`](#childProcessResult).
```

Comment:
I am wondering whether we should split this section into two?
  - $\`command\`, which executes a command
  - `$(options)`, which returns a new `$` with specific default `options`

This might also make it clearer that users do not need to call `$(options)` to execute a command.

Reply:
Good call 👍 Here's my first pass at breaking the `$`, `$.sync`, and `$(options)` APIs into dedicated sections: https://github.com/sindresorhus/execa/pull/510/commits/33f4d065a0d02172ae23e658ce05126d7a679e89